File: Python/functions/no checking/karpekar.py
#Karpekar no.
def check(n):
    sq=n**2
    c=len(str(n))
    if n==(sq%(10**c))+(sq//10**c):
        return True
    else:
        return False
n=int(input("Enter the no. = "))
if check(n):
    print("Karpekar no.")
else:
    print("Not Karpekar No.")

File: C Programms/No. Checking/karpekar.c
//WAP to check Karpekar no.
#include<stdio.h>
void main()
{
    int n,sq,p=1,s,x;
    printf("Enter the no. = ");
    scanf("%d",&n);
    for(sq=n*n,x=n;n!=0;n/=10)
        p*=10;
    s=sq%p + sq/p;
    (s==x)?printf("%d is a Karpekar No.",x):printf("%d is not a Karpekar No.",x);
}